package.json

```diff
@@ -1,47 +1,77 @@
 {
   "name": "ember-cli-webcomponents-bundler",
   "version": "0.0.0",
-  "description": "The default blueprint for ember-cli addons.",
+  "description": "Allows to use Web Components using ES modules in your Ember app",
   "keywords": [
-    "ember-addon"
+    "ember-addon",
+    "web components",
+    "ES6 modules",
+    "bundle"
   ],
-  "repository": "",
+  "repository": "https://github.com/BBVAEngineering/ember-cli-webcomponents-bundler.git",
   "license": "MIT",
-  "author": "",
+  "author": "BBVAEngineering",
   "directories": {
-    "doc": "doc",
     "test": "tests"
   },
+  "files": [
+    "index.js",
+    "config/babel.js",
+    "config/output.js",
+    "config/rollup.js",
+    "utils/**"
+  ],
   "scripts": {
     "build": "ember build",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint --max-warnings 0 ./*.js addon app config tests",
+    "lint:js": "eslint --fix --max-warnings 0 ./*.js config tests",
     "start": "ember serve",
-    "test": "COVERAGE=true ember test",
-    "test:all": "ember try:each",
-    "check-coverage": "istanbul check-coverage coverage/coverage-final.json --statements 100 --functions 100 --branches 100 --lines 100",
+    "test": "npm-run-all test:*",
+    "test:ember": "COVERAGE=true ember test",
+    "test:node": "nyc mocha --recursive 'node-tests/**/*-test.js'",
+    "clean-coverage": "rm -rf coverage",
+    "merge-coverage": "cp coverage/ember/coverage-final.json coverage/ember.json && cp coverage/node/coverage-final.json coverage/node.json && nyc merge coverage coverage/coverage-final.json",
+    "check-coverage": "istanbul check-coverage coverage/coverage-final.json --statements 85 --functions 85 --branches 85 --lines 85",
```

Comment:
100? 😄

Reply:
If I use 100%, I'll have to test things like that:

```js
const options = baseConfig[this.name] || {};
env = env || process.env.EMBER_ENV;
```

Do you really think that this kind of test are useful?
For me they are a waste of time that only adds more noise to the tests.

Reply:
Then set it as the current coverage at least, in order to force check when coverage falls from the threshold.

Reply:
Ok... 100% 9c01149

Reply:
🔝